src/CUITeTest/SearchConfigurations/BaseConfiguratorTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿using CUITe.SearchConfigurations;
+using Microsoft.VisualStudio.TestTools.UITesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CUITeTest.SearchConfigurations
+{
+    using System;
```

Comment:
Yes. I'm wondering if we should check in your copy of the stylecop settings file that we should use?